### PR TITLE
Add thumbnail version for image

### PR DESCRIPTION
**Pull request by fbernutz**
### Motivation:

After testing how the new blog post looks on the Swift.org website, I noticed that the sketchnote takes a long time to load, which is not a good user experience. 

Affected pages:
- https://www.swift.org/blog/
- https://www.swift.org/blog/swift-language-updates-from-wwdc22/

To improve the user experience for everyone but especially for users with low band width, there's now a smaller thumbnail version displayed and a link to the full resolution image.

### Modifications:

- Add thumbnail image with lower resolution.
- Show thumbnail version in blog post
- Add link to full resolution asset

### Result:

The website will load quicker and use less band width while visitors of the site are still able to inspect the full resolution when using the link.

Screen Shot 2022-07-06 at 11 52 28

### CODE REVIEW

File: _posts/2022-07-01-swift-language-updates-from-wwdc22.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ title: "Swift language announcements from WWDC22"
 author: [fbernutz, hishnash, natpanferova]
 ---
 
-![Sketchnote summary for Swift language announcements from WWDC22 blog post](/assets/images/swift-language-updates-from-wwdc22-blog/wwdc22-swift-updates-sketch.jpeg)
+![Sketchnote summary for Swift language announcements from WWDC22 blog post](/assets/images/swift-language-updates-from-wwdc22-blog/wwdc22-swift-updates-sketch-thumbnail.jpeg)
```
Comment (fbernutz) [inline _posts/2022-07-01-swift-language-updates-from-wwdc22.md:9]: by default the thumbnail image is displayed which has a size of 400KB instead of 2.5MB

**Comment by alexandersandberg** [comment]
Good idea! Is it possible that we can compress the thumbnail even further without losing quality? I quickly tried running it through [tinypng.com](tinypng.com) and got it to ~200KB—although _with_ slightly lesser quality.

**Comment by fbernutz** [comment]
@alexandersandberg you're right! I made the thumbnail even smaller. It's a compromise between quality and size and it's now 269KB large.

**Review by alexandersandberg — APPROVED** [review]
Perfect! 🙂

**Comment by alexandersandberg** [comment]
@swift-ci test

**Review by daveverwer — APPROVED** [review]
Great catch @fbernutz! Thank you.

**Comment by daveverwer** [comment]
@swift-ci test

**Pull request merged**